test/contrib/rails/controller_test.rb

```diff
@@ -108,7 +108,7 @@ class TracingControllerTest < ActionController::TestCase
     assert_equal(1, span.status, 'span should be flagged as an error')
     assert_equal('ZeroDivisionError', span.get_tag('error.type'), 'type should contain the class name of the error')
     assert_equal('divided by 0', span.get_tag('error.msg'), 'msg should state we tried to divided by 0')
-    assert_nil(span.get_tag('error.stack'))
+    assert_empty(span.get_tag('error.stack'))
```

Comment:
why it has been changed? `assert_nil` ensures that `get_tag()` returns `nil`. `assert_empty` does the same? because we must not have the `error.stack` set, otherwise if we have (for instance) `""`, it ends in sending wrong data.